CHANGELOG.md

```diff
@@ -1137,4 +1137,8 @@ should use 4.0.1-alpha.0 for testing.
 
 ## [Unreleased]
 
+### Added
+
+#### web3-core
+
```

Comment:
Ran `yarn changelog sync` to make this change as current `4.x` `CHANGELOG.md` is missing these headers

![image](https://user-images.githubusercontent.com/20375610/214470523-54e1ecf1-2460-47c7-84a1-242dda216f94.png)